### ISSUE TITLE
setup

### ISSUE THREAD

**Issue by njamescouk** [swcarpentry/sql-novice-survey#144]
the page http://swcarpentry.github.io/sql-novice-survey/guide/ section SQLite Setup clones the git repo, which seems unnecessary if this page is intended for learners to setup, and in any event breaks down for me at the line `sqlite3 survey.sqlite .read bin/create-db.sql`

Am I missing something here?

**Comment by debpaul** [comment]
doesn't work for me either. I'm trying to understand what the issue is. I've tried installing software on two different computers. Win 10, surface pro 4, and a surface pro 3. FIRST, IF I install all software for an SwC workshop (where we're doing R, not Python) http://idigbio.github.io/2016-12-03-tdwg-costa-rica/, and then download the survey.db file from http://swcarpentry.github.io//sql-novice-survey/, and THEN do `sqlite3 survey.db` at the git bash prompt, it still doesn't work. NEXT, IF I do run the `git clone http://github.com/swcarpentry/sql-novice-survey.git` AND THEN `sqlite3 survey.db`  at the git bash prompt - it works. SO, does this mean I'm missing a step somewhere? OR Will all students need to do the `git clone` step? (not mentioned in any part of the lessons)? Thanks for clarification!

**Comment by remram44** [comment]
The command `sqlite3 survey.sqlite .read bin/create-db.sql` is wrong since quoting is necessary.

```console
$ sqlite3 survey.sqlite .read bin/create-db.sql
Usage: .read FILE
(fails)

$ sqlite3 survey.sqlite '.read bin/create-db.sql'
(works)
```

**Comment by njamescouk** [comment]
bit late with this! but appears to be resolved.

**Comment by remram44** [comment]
Fixed by #196
